src/client/components/UploadImage/UploadImage.jsx

```diff
@@ -20,10 +20,10 @@ export default function CreateSessionImage() {
   return (
     <Box sx={{paddingLeft: 10}}>
         {selectedImage ? (
-          <img className="create-session-image" src={selectedImage} alt="Selected"/>
+          <img className="upload-image" src={selectedImage} alt="Selected"/>
```

Comment:
Think we could make these a squircle, just so it matches with the rest of our UI? Your call though, I'm good with either. 

https://blog.prototypr.io/squirclemorphism-in-user-interfaces-9a3f2c157c08

Reply:
Umm, I tried making the pic a squircle but it doesn't let me. Maybe you can try? (`border-radius` doesn't work) Will invest time in this if required but doesn't look like a big deal to me.